CommonSuite/VINDecoder.cs

```diff
@@ -659,22 +655,5 @@ public string GearboxDescription
             get { return _gearboxDescription; }
             set { _gearboxDescription = value; }
         }
-
-        private string _checksumText;
-
-        public string ChecksumText
-        {
-            get { return _checksumText; }
-            set { _checksumText = value; }
-        }
-
-        private bool _validChecksum;
```

Comment:
Hi,

Id like to have this (IsChecksumValid) readded to CarInfo if possible so that I can asset for that in the unit test.

The units tests I can help you with its the SuiteTest project that I did not add to the T7 solution.

/Mattias